.github/workflows/welcome-newcomers.yaml

```diff
@@ -1,11 +1,11 @@
-name: first-interaction
+name: Say Hello to Firsttimers
 
 on:
   issues:
     types: [opened]
-  pull_request:
-    branches: [main]
+  pull_request_target:
```

Comment:
Would be good to add a comment referencing the blog and explaining that this is safe because we don't check out any code.